src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-how-nrql-works.mdx

```diff
@@ -0,0 +1,295 @@
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+These are the basic rules for NRQL, and we have more information on [NRQL syntax and functions](/docs/query-data/nrql-new-relic-query-language/getting-started/nrql-syntax-components-functions) to help you build your queries.
+
+## Data exploration [#explore-data]
+
+One of the best ways to learn how to use NRQL is to access one of the [query tools](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language#where) and use it to explore your data. Here's an example of exploring your data using the [query builder](/docs/chart-builder/use-chart-builder/get-started/introduction-chart-builder) and the suggested entries from the interface. 
+
+The query begins with `FROM` followed by space. The interface suggests available types of data, and `Transaction` is chosen from that list.
+
+<img
+  title="nrql_query01.png"
+  alt="A screenshot of a NRQL query beginning with From"
+  src={queriesnrqlEventDefinitionsQueryBuilder}
+/>
+
+Next, an [attribute](/docs/using-new-relic/welcome-new-relic/get-started/glossary#attribute) is chosen using `SELECT`, and the query looks as follows:
+
+```sql
+FROM Transaction SELECT
+```
+
+Pressing space again causes the interface to suggest available attributes. `appId` is chosen.
+
+<img
+  title="nrql_query02.png"
+  alt="A screenshot of a NRQL query using the previous screenshot and adding Select"
+  src={queriesnrqlAttributeDefinitionsQueryBuilder}
+/>
+
+The result is a very basic NRQL query using the required clause and statement (`FROM` and `SELECT`), and it provides a list of APM transactions and the associated `appId` for each, as shown below.
+
+<img
+  title="basic_nrql_example.png"
+  alt="A screenshot of a basic NRQL query, From Transaction Select appId"
+  src={queriesNrqlBasicExample}
+/>
+
+<Callout variant="tip">
+Don't be afraid to play with your data! You won't break anything using any of our query interfaces, so feel free to explore early and often!
+</Callout>
+
+Another great way to explore your data is to go into any existing dashboards you have, click **View query**, and run your chart in the query builder.
+<img
+  title="new-relic-view-chart-nrql-query.png"
+  alt="New Relic view chart NRQL query"
+  src={queriesnrqlViewQueryforChart}
+/>
+
+<figcaption>
+  Charts built with NRQL will have **View query** as an option. You can then edit and customize that query to see how your changes affect the resulting visualization.
+</figcaption>
+
+## NRQL query examples [#examples]
+
+Here's an example of a slightly more in-depth NRQL query of `Transaction` data, which is reported by [APM](/docs/apm). For this query, `Transaction` is the chosen data type, `Select` is used to determine the average duration, results are grouped by appName using `Facet`, and `Timeseries` is used to display the data over an automated timespane.
```

Comment:
This is an amazing example! I would separate this out into a bulleted list for easier reading though

Reply:
Thank you and good call, I've made the change